lib/commands/serve.js

```diff
@@ -38,10 +39,11 @@ module.exports = Command.extend({
   ],
 
   run(commandOptions) {
+    commandOptions.liveReloadPort = commandOptions.port;
```

Comment:
If an explicit port was provided on the command line via `--live-reload-port` option, we should warn